bite-size-notes/week-4-d-Example of LQR-name.md

```diff
@@ -0,0 +1,350 @@
+# Objectives: 
+1. Motivation of LQR
+2. Summarize the variation of LQR and corresponding Recati equation and optimal control. 
+3. Using ode45 to solve Reccati equation.
+# Introduction: 
+LQR (Linear Quatratic Regulater) is the a optimal solution to a full state feedback control to minimine the state error and control efforts. Let's break down this sentence to understand what is LQR ? 
+## What is a fullstate feedback control.
+We want to stabilize a system and drive all state to 0. A standard full state feed back control can achieve this goal.(There are other variation which are out of the scope) \
+A standar full feedback controle has the block diagram. 
+ ![fullfeedback](https://github.com/p8410077/AA548-spr2024/assets/11802603/bea8ed3a-5062-4d60-93c9-b213d4d856d5)
+
+By using the full state control, we can using the method called pole placement, literally, we are putting the poles of the system at the desired location on the complex plane.
+
+## what is pole placement ?
+The open loop state space representices of the system is:
+
+$$
+	\dot{x}(t) = Ax(t)+ Bu(t) \\
+$$
+
+$$
+	y(t) = Cx(t) +Du(t) \
+$$
+
+By finding the location of the poles of matrix A, we can understand if the system is stable. We can design a controller that stabilize the system, if the system is unstable, or drive the system to converge faster, if the system is stable. 
```

Comment:
The singularity of "location" and plurality of "poles" implies that the poles of $A$ are all the same.